modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowTaskManagerImplTest.java

```diff
@@ -825,6 +830,54 @@ public void testDeleteUserWithWorkflowTaskAssigned() throws Exception {
 		_deactivateWorkflow(BlogsEntry.class.getName(), 0, 0);
 	}
 
+	@Test
+	public void testFailedScriptedAssignment() throws Exception {
+		String workflowDefinitionName = "Failed Scripted Single Approver";
```

Comment:
@pedro-oliveira446 rename to `Broken Scripted Assignment Workflow Definition`

Reply:
Done!